lychee-lib/src/utils/request.rs

```diff
@@ -125,24 +125,22 @@ fn truncate_source(source: &InputSource) -> InputSource {
 /// it will not be added to the `HashSet`.
 pub(crate) fn create(
     uris: Vec<RawUri>,
-    input_content: &InputContent,
+    source: &InputSource,
```

Comment:
this was just a simplification to make it easier to write tests, since only `input_content.source` is used by this function